api/iteration.go

```diff
@@ -2,52 +2,74 @@ package api
 
 import (
 	"fmt"
+	"io/ioutil"
 	"path/filepath"
 	"strings"
 )
 
 const (
-	msgUnidentifiable = "unable to identify track and problem"
+	errUnidentifiable = "unable to identify track and problem"
```

Comment:
Since we know these are errors and for the most part we are returning them with `fmt.Errorf(error)` why not just create them with `errors.New(...)` and return the error itself without `fmt.Errorf`?